src/graph-view-node.js

```diff
@@ -376,7 +376,7 @@ class GraphViewNode {
             items: this._graphView._parent._initialiseNodeContextMenuItems(this.nodeData, items)
```

Comment:
Looks like this should be updated too.

Reply:
Oh, good catch! I was trusting VS Code to rename all instances. I guess that's not so reliable with JS vs TS. Anyway, fixed!